### PR TITLE
Adding hints for radio button options

### PR DESCRIPTION
**Pull request by waqarnazir-ofgem**

### CODE REVIEW

File: GovUkHtmlHelperExtensions.cs [hunk 1]
```diff
@@ -254,15 +254,18 @@ public static class GovUkHtmlHelperExtensions
             Expression<Func<TModel, TEnum?>> propertyLambdaExpression,
             FieldsetViewModel fieldsetOptions = null,
             HintViewModel hintOptions = null,
+            Dictionary<TEnum, HintViewModel> radioHints = null,
             string classes = null)
             where TModel : class
             where TEnum : struct, Enum
+
```
Comment (DanCorder) [inline GovUkHtmlHelperExtensions.cs]: Please remove this empty line

File: HtmlGenerators/RadiosHtmlGenerator.cs [hunk 2]
```diff
@@ -37,6 +38,12 @@ internal static class RadiosHtmlGenerator
                 {
                     bool isEnumValueCurrentlySelected = selectedValue.HasValue && enumValue.Equals(selectedValue.Value);
                     string radioLabelText = GovUkRadioCheckboxLabelTextAttribute.GetLabelText(enumValue);
+                    HintViewModel itemHint = null;
+
+                    if (radioHints != null)
```
Comment (DanCorder) [inline HtmlGenerators/RadiosHtmlGenerator.cs]: I think this could be `radioHints?.TryGetValue(enumValue, out itemHint);`

**Review by DanCorder — CHANGES_REQUESTED** [review]

File: GovUkHtmlHelperExtensions.cs (hunk 1 above)
Comment (waqarnazir-ofgem) [reply GovUkHtmlHelperExtensions.cs]: Done, thanks

File: HtmlGenerators/RadiosHtmlGenerator.cs (hunk 2 above)
Comment (waqarnazir-ofgem) [reply HtmlGenerators/RadiosHtmlGenerator.cs]: Done, thanks

**Review by DanCorder — APPROVED** [review]

**Pull request merged**